kratos/solving_strategies/schemes/residual_based_bdf_custom_scheme.h

```diff
@@ -669,9 +704,7 @@ class ResidualBasedBDFCustomScheme
             "name"                  : "ResidualBasedBDFCustomScheme",
             "domain_size"           : 3,
             "integration_order"     : 2,
-            "variable"              : ["DISPLACEMENT"],
-            "first_derivative"      : ["VELOCITY"],
-            "second_derivative"     : ["ACCELERATION"]
+            "solution_variables"    : ["DISPLACEMENT"]
```

Comment:
I think you misunderstood me so far, the input is reduced now, but like this it will not compile
can you please update the rest?